### DISCUSSION TITLE
Advice: Authoring different kinematics (Delta robot)

### DISCUSSION THREAD

**Discussion by iLoveAndyBaker** [grblHAL/core#342] [Q&A]
I've built a "delta" robot, and I have code that works to do the kinematics - Give it XYZ (and the geometric parameters of the physical parts of the rig) and it spits out motor angles. Excellent! Now what?!?!

[https://github.com/tinkersprojects/Delta-Kinematics-Library](url)

Please forgive me for not being a tremendous programmer. I can do a bit here and there, but it's not my area of expertise. Making the mechanical parts is what I'm good at. 

Thank you!

**Comment by terjeio** [comment]
[This comment](https://github.com/grblHAL/core/issues/341#issuecomment-1675372776) may contain some guidance.

**Comment by terjeio** [comment]
Continues [here](https://github.com/grblHAL/core/issues/346).
